uqbar-project/wollok
# Upgrade eclipse neon xtext

Updates eclipse to 4.6 neon
Updates xtext to 2.10.0

To test this branch you need to:

- Download a new eclipse version NEON and package "for DSLs". from here http://www.eclipse.org/downloads/packages/eclipse-ide-java-and-dsl-developers/neonr
- Import all wollok projects. It will automatically detect that it needs some plugins to be able to build with m2e. Accept to install all extensions.
- Then open the file "org.uqbar.project.wollok.targetplatform.target". And press the link that appears on the top-right "Set as Target Platform" that will rebuild all your wollok projects against that target platform (a definition of an eclipse product with its plugins) instead of against your running eclipse.

That should make it compile.
If not, then maybe try right-click Maven -> "Update Projects".
